DEVELOPMENT.md

```diff
@@ -42,15 +42,29 @@ Install these tools:
     export KO_DOCKER_REPO="kind.local"
     ```
 
-1. Install knative-eventing core and cert-manager and wait for them to be ready:
+1. Install knative-eventing core and cert-manager, wait for them to be ready, and enable new-apiserversource-filters:
```

Comment:
Rather than have this code in the DEVELOPMENT.md file, why not refer them to the `hack/quick-install.sh` script? Isn't this the same thing?

Reply:
The hack/quick-install is used on the CI. We may have different instructions for CI than for development, as we mentioned yesterday in the meeting (CI to test production procedure).

After we make changes for production release at #277 , maybe we can see if it makes sense to replace some DEVELOPMENT steps by the hack/quick-install.sh

Reply:
I'm going to resolve this thread for now and we can discuss doing this after #277